src/pages/components/menu-buttons/usage.mdx

```diff
@@ -153,17 +162,10 @@ implementation in other contexts if needed.
 
 </InlineNotification>
 
-| Size        | Height (px / rem) |
-| ----------- | ----------------- |
-| Extra small | 24 / 1.5          |
-| Small       | 32 / 2            |
-| Medium      | 40 / 2.5          |
-| Large       | 48 / 3            |
-
 <Row>
-<Column colLg={8}>
+<Column colLg={12}>
 
-![Menu height](images/usage-menu-button-height.png)
+![Combo button height](images/usage-combo-button-height.png)
```

Comment:
I made a couple comments in your Figma file on some of the images, just some small styling things, otherwise they look good!